hydra-node/test/Hydra/BehaviorSpec.hs

```diff
@@ -587,7 +593,11 @@ createHydraNode ledger signingKey otherParties outputs outputHistory connectToCh
   chainComponent connectToChain $
     HydraNode
       { eq
-      , hn = Network{broadcast = const $ pure ()}
+      , hn =
+          Network
+            { broadcast = const $ pure ()
+            , modifyPeers = error "createHydraNode.modifyPeers: unused"
```

Comment:
Is this whole function even used? If not, let's remove it. If yes, this still should be avoidable by restructuring this and ConnectToChain

Reply:
What 'whole function'  refers to here? `modifyPeers`?

Reply:
created a no network fixture definition to handle this scenarios